docs/configuration.md

```diff
@@ -314,6 +314,32 @@ The `hidden` value is an optional value that can be used to not list
 this identity provider in the list of possible identity providers when
 performing an interactive login.
 
+### Keycloak OpenID Connect
+```yaml
+- type: keycloak
+  domain: example
+  client-id: 483156874216
+  client-secret: 32hf3uhud23dS@#e
```

Comment:
Secret should be an optional parameter. Some businesses 'trust' the client and do not want the secret to be put into 'untrusted' areas, thus just allow clients to connect without a secret.

Reply:
Latest commit should make it optional. Note, I don't have a way to personally test this setup in production as my organization requires a secret.